ansible/library/extract_log.py

```diff
@@ -116,7 +116,6 @@ def extract_number(s):
     else:
         return int(ns[0])
 
-
```

Comment:
Minor: two places where line breaks are removed. Double line break is a pip8 suggested format.

Reply:
Updated